README.md

```diff
@@ -49,7 +49,7 @@ SeaTunnel will do its best to solve the problems that may be encountered in the
 
 ## Workflow of SeaTunnel
 
-![seatunnel-workflow.svg](https://raw.githubusercontent.com/apache/incubator-seatunnel-website/main/static/image/seatunnel-workflow.svg)
```

Comment:
thanks for your contribution,
This is the official material library of github, we don't need to modify it.